packages/server/src/integrations/airtable.ts

```diff
@@ -63,7 +63,11 @@ module AirtableModule {
         type: QueryTypes.FIELDS,
         customisable: true,
         fields: {
-          id: {
```

Comment:
Ah good catch. It would be a breaking change, so maybe not worth the rename. 
If I was to add an additional field and do a check would that do? Something like:
```js
const { table, recordID, id, json } = query
if (!recordID) {
  recordID = id;
}
```

Happy to revert as well, just thought it would be worth being explicit about which ID is being requested.

Reply:
Yes you could do it that way for backwards compat - I think the bigger point is that we possibly need a `displayName` attribute that can be attached to the property, so that the same key (`id` in this case) can still be used, but you can change what it is displayed as. This would also allow it to be displayed as `Record ID` instead of `RecordID` in the frontend.

This might not actually be a super difficult change and probably worth doing - but I think in this case changing the props name is probably not worth the hassle - it would be better to just support changing what it is displayed as.

Reply:
@mike12345567 Someone had already thought of this! Now using the display field.